pages/index.tsx

```diff
@@ -1,16 +1,49 @@
 import { LandingHero } from "@components/LandingHero";
+import { LandingHeroBackgroundMap } from "@components/LandingHeroBackgroundMap";
 import { LandingHowItWorks } from "@components/LandingHowItWorks";
 import { LandingLabAbout } from "@components/LandingLabAbout";
 import { LandingProjectAbout } from "@components/LandingProjectAbout";
 import { LandingProjectsSlider } from "@components/LandingProjectsSlider";
-import { fakeProjects } from "@components/LandingProjectsSlider/LandingProjectsSlider.stories";
-import { FC } from "react";
+import { LandingStatsSection } from "@components/LandingStatsSection";
+import {
+  getLandingStats,
+  LandingStatsReturnType,
+} from "@lib/requests/getLandingStats";
+import { fakeCuratedProjects } from "@mocks/data";
```

Comment:
Hmm, I'm having doubts about mock data making it to production, even if we are in a very early stage. 

I would suggest defining the curated projects as a subset of our already existing mock projects - say simply the first 3. And then import a yet imaginary hook here `useCuratedProjects`. Of course we need to find a way to highlight curated projects in our production database, but for now we could just say we use the first 3 as well. What do you think @vogelino?

Reply:
Good point! I will address that. I know that this isn't ideal. But we need real data anyway. I agree that using the first three projects might be a valid solution, however, we should create them as well. Right now, the projects in our dev DB are worse than our mock data. Neither solutions seem to be great. It will probably be better when @lucasoeth has filled the v3 DB with nice-looking fake content.

Reply:
Agree. We need realistic data in the new DB!

Reply:
See bd42a67